cmake/SetupLicenseInfo.cmake

```diff
@@ -0,0 +1,74 @@
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+set(_LIST_OF_3PLS
+  Blaze
+  Boost
+  Brigand
+  Catch2
+  Gsl
+  Hdf5
+  Libbacktrace
```

Comment:
[optional] If you're feeling fancy, you can exclude optional dependencies that weren't enabled.  I don't think always including them is a problem, though.

Reply:
I thought about this before already and then decided I was too lazy 🙃